.travis.yml

```diff
@@ -11,7 +11,3 @@ script: "./.travis/travis-run.sh"
 
 after_success:
   - "./.travis/travis-deploy.sh"
-
-branches:
-  only:
-  - master
```

Comment:
Please remove this change.

Reply:
Yeah, I didn't find another way to trigger build on different branches than master on my side, so I had to do that to build a build, but I just reverted it.